### PR TITLE
Feat/pwn install fn

### PR DESCRIPTION
**Pull request by valentinnavalos**

### CODE REVIEW

File: packages/web-app/src/utils/config.ts [hunk 1]
```diff
@@ -61,25 +73,29 @@ export const PLUGIN_IDS: { [K in SupportedNetwork]: PluginsDeployment } = {
         creditDelegation: "my-cool-plugin-6016",
         subgovernance: "my-cool-plugin-6002",
         uniswapV3: "my-cool-plugin-6017",
-        vault: "my-cool-plugin-6005"
+        vault: "my-cool-plugin-6005",
+        pwn: "",
```
Comment (ialberquilla) [inline packages/web-app/src/utils/config.ts]: Address is missing

Comment (ialberquilla) [reply packages/web-app/src/utils/config.ts]: Should be 0x88c107d4271dfda5bc173049db9005b54fd56541

**Review by ialberquilla — APPROVED** [review]

File: packages/web-app/src/utils/config.ts (hunk 1 above)
Comment (valentinnavalos) [reply packages/web-app/src/utils/config.ts]: But this empty string, at line 77, is on the IDs array. Not in the addresses array. Are you suggesting that we should also include that address as an ID?

Comment (ialberquilla) [reply packages/web-app/src/utils/config.ts]: Yes, line 77 should be the name of the plugin  registered in the plugin preprocessor, for pwn should be my-cool-plugin-6020

**Pull request merged**